### ISSUE TITLE
is it possible to rename the branch created in fork repo?

### ISSUE THREAD

**Issue by Vijayalakshminp1985** [jayarajvemula/bits-osseproject-group52#7]
While trying to create a pull request along with branch creation and new file update, i used the default filename for branch name and did not update the proper branch name.. is it possible to correct the same?

**Comment by jayarajvemula** [comment]
Yes please . Create another pull request and make necessary change. We could also revert the same , but not picking that path here.
